README.md

```diff
@@ -4,7 +4,7 @@
 
 A JavaScript / Python / PHP library for cryptocurrency trading and e-commerce with support for many bitcoin/ether/altcoin exchange markets and merchant APIs.
 
```

Comment:
I only changed this line, the rest was generated from the change in export-exchanges

Reply:
@pcriadoperez did you change only ccxtProBadge inside export-exchanges?  If so we don't need to push the README file

Reply:
Yes, that updated most of the README file with the exception of line 7 which I had to update manually, that's why I included I included the README file in the PR